docs/developer-guide/ksqldb-reference/select-pull-query.md

```diff
@@ -14,98 +14,130 @@ Synopsis
 
 ```sql
 SELECT select_expr [, ...]
-  FROM aggregate_table
-  WHERE key_column=key [AND ...]
-  [AND window_bounds];
+  FROM table
+  [ WHERE where_condition ]
+  [ AND window_bounds ];
 ```
 
 Description
 -----------
 
-Pulls the current value from the materialized table and terminates. The result
-of this statement isn't persisted in a {{ site.ak }} topic and is printed out
-only in the console.
-
-Pull queries enable you to fetch the current state of a materialized view.
-Because materialized views are incrementally updated as new events arrive,
-pull queries run with predictably low latency. They're a great match for
-request/response flows. For asynchronous application flows, see
+Pulls the current value from the materialized view and terminates. The result
+of this statement is not persisted in a {{ site.ak }} topic and is printed out
+only in the console. Because materialized views are incrementally updated as new 
+events arrive, pull queries run with predictably low latency. They are a great match 
+for request/response flows. For asynchronous application flows, see 
 [Push Queries](select-push-query.md).
 
-Execute a pull query by sending an HTTP request to the ksqlDB REST API, and
+You can execute a pull query by sending an HTTP request to the ksqlDB REST API, and

```

Comment:
It streams back a chunked response.  Technically, it's a single response, though maybe it's a little misleading, since I think that wording was intended to mean it's returned in one fell swoop (one chunk!).  Maybe we want to change the description to either say "Streams back a chunked response as rows are fetched" or just make it a little more vague so that it's a little less tied to an implementation by saying "the API responds with resulting rows".